ts/nni_manager/config/aml/amlUtil.py

```diff
@@ -44,7 +44,16 @@
             print('tracking_url:' + run.get_portal_url())
         elif line == 'stop':
             run.cancel()
-            exit(0)
+            loop_count = 0
+            status = run.get_status()
+            # wait until the run is canceled
+            while status != 'Canceled':
+                if loop_count > 5:
+                    print('stop_result:failed')
+                    exit(0)
+                loop_count += 1
+                time.sleep(500)
+            print('stop_result:success')
```

Comment:
do we need `exit(0)` in this place?

Reply:
sure, generally it will exit automatically when main process stop, add exit here to avoid corner case.